README.md

```diff
@@ -1 +1,13 @@
 # WSL
+# Text file information reader 
+This program will display the following information about a given text file:
+-File size
+-Number of characters
+-Creation date 
+-Last modification date
+To achieve this, we will first need to import the following libraries:
+-os
+-time
+-path
+We also need to get the path of a file. To do this, we can use the "realpath (filename)" command.
+Then, we will need to get the file siz,e number of characters, and creation/modification dates. For this, we will use the libraries we have imported.
```

Comment:
Don't try to solve the problem in this text. Think of it as action items --> smaller steps I need to implement to support functional requirements (see previous comment).

From the function requirements I can think of the following steps:
 * implement reading parameters from the command line (because you need to read filepath)
 * load a file from provided filepath (no need to go any further)
 * read file metadata (now we get to interesting parts) --> we need file creation, modification dates, possibly file size?
 * read file contents --> get number of characters, maybe file size??

This list can be extended whenever you want to. But it is important to always write down concrete functionality that you want to implement and keep it as simple as possible.